generated/nifake_non_ivi/nifake_non_ivi.proto

```diff
@@ -36,9 +36,12 @@ service NiFakeNonIvi {
   rpc SetMarbleAttributeInt32(SetMarbleAttributeInt32Request) returns (SetMarbleAttributeInt32Response);
 }
 
-enum MarbleAttributes {
-  MARBLE_ATTRIBUTE_UNSPECIFIED = 0;
+enum MarbleAttributesInt32 {
+  MARBLE_INT32_ATTRIBUTE_UNSPECIFIED = 0;
```

Comment:
If we keep the enum name as `MarbleAttributesInt32` the UNSPECIFIED value should probably be `MARBLE_ATTRIBUTE_INT32_UNSPECIFIED`. I don't really like that, but it doesn't matter much as long as it's just unspecified. (Seem comments in `proto_helpers.mako` on how this works).